ex36 - without is_game_continue.py

```diff
@@ -0,0 +1,206 @@
+from sys import exit
+
+
+def start():
+	print("You don't remember anything. You are fallen.")
+	print("You don't know anywhere to go but the path ahead.")
+	print("You begin to step forward into the forest...")
+	# to Chest zone
+	got_chest_key = chest()
+	# to Crow zone
+	got_crow_key = crow()
+	# to The Great Wall
+	great_wall(got_chest_key, got_crow_key)
+
+	# Good ending?
+	print("Game over.")
+	exit(0)
+
+# The Chest zone
+def chest():
+
+	print("You see a shiny chest lying near a bush ahead.")
+	print("Who knows what valuable stuff it would contain?")
+
+	# Chest Key taken?
+	chest_key_obtained = False
+
+	while not chest_key_obtained:
+		## Remind player of choices
+		show_chest_menu()
+
+		chest_choice = input("> ")
+	
+		if chest_choice == "1":
+			print("\n================================================")
+			print("You look carefully at the lock area of the chest.", end=' ')
+			print("It is rusted badly. \nAnd very FRAGILE.")
+			print("================================================\n")
+			
+		elif chest_choice == "2":
+			chest_key_obtained = open_chest()
+		else:
+			print("Quit too soon?")
+			dead("You died of giving up your purpose.")
+	# Got the key
+	return chest_key_obtained
+
+def open_chest():
+	print("Alright! Let's open this chest.")
+	
+	while True:
+		## Remind player of choices
+		show_open_chest_menu()
+
+		open_choice = input("> ")
+	
+		if open_choice == "1":
+			print("\n================================================")

```

Comment:
This long line is repeated everywhere, use a constant for it or put the print statement in a function

Reply:
I think this can be deleted from code.
We don't have a proper UI for this project anyway.